plugins/language_perl.lua

```diff
@@ -266,26 +263,10 @@ syntax.add {
     ["each"] = "keyword",
     ["lcfirst"] = "keyword",
     ["setnetent"] = "keyword",
-    ["while"] = "keyword",
-    ["for"] = "keyword",
-    ["if"] = "keyword",
-    ["else"] = "keyword",
-    ["elsif"] = "keyword",
-    ["unless"] = "keyword",
-    ["no"] = "keyword",
-    ["new"] = "keyword",
-    ["do"] = "keyword",
-    ["__PACKAGE__"] = "keyword",
-    ["warnings"] = "keyword2",
-    ["strict"] = "keyword2",
-    ["eq"] = "operator",
-    ["ne"] = "operator",
-    ["lt"] = "operator",
-    ["gt"] = "operator",
-    ["le"] = "operator",
-    ["ge"] = "operator",
-    ["cmp"] = "operator",
-    ["STDERR"] = "keyword2",
-    ["STDOUT"] = "keyword2"
```

Comment:
What's the reason for the removal of these symbols?

Reply:
Not sure how these got removed, I'll clean it up and squash.